airflow/contrib/executors/mesos_executor.py

```diff
@@ -63,6 +63,9 @@ def __init__(self,
         self.task_mem = task_mem
         self.task_counter = 0
         self.task_key_map = {}
+        if configuration.get('mesos', 'DOCKER_IMAGE_SLAVE'):
+            self.mesos_slave_docker_image = \
```

Comment:
Can you make this cleaner? Ie. without the `\` for the multiline. Maybe use parenthesis?